critiquebrainz/frontend/templates/review/delete_comment.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+
+{% set entity = review.entity_id | entity_details(type=review.entity_type) %}
+{% if review.entity_type == 'release_group' %}
+  {% set entity_title = entity['title'] | default(_('[Unknown release group]')) %}
+{% elif review.entity_type == 'event' %}
+  {% set entity_title = entity['name'] | default(_('[Unknown event]')) %}
+{% elif review.entity_type == 'place' %}
+  {% set entity_title = entity['name'] | default(_('[Unknown place]')) %}
+{% endif %}
+
+{% block title %}
+  {{ _('Edit comment on %(user)s\'s review of "%(entity)s"', entity=entity_title, user=review['user'].display_name) }} - CritiqueBrainz
+{% endblock %}
+
+{% block content %}
+  <h2>{{ _('Deleting comment!') }}</h2>
+  <hr />
+  <form method="POST" role="form">
+    <div class="form-group">
+      <p class="lead">{{ _('Are you sure you want to delete your comment on %(user)s\'s review of "%(entity)s"?', entity=entity_title, user=review['user'].display_name) }}</p>
```

Comment:
I would like to see the content of the comment as well here.

Reply:
Delete page now shows the comment user is going to delete.